ITRStartTask/AppLibrary/ApiService/Operations/SyncOperation.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AsyncOperation.swift
+//  ITRStartTask
+//
+//  Created by Homyakov, Ilya2 on 22/08/2018.
+//  Copyright © 2018 Homyakov, Ilya2. All rights reserved.
+//
+
+import UIKit
+
+class SyncOperation: Operation {
+    override var isAsynchronous: Bool {
+        return false
+    }
+
+    private var isFinishedValue: Bool = false
+    private var isExecutingValue: Bool = false
+
+    override var isFinished: Bool {
```

Comment:
Read docs and make corrections with Sync tasks properties overriding

Reply:
fixed